pkg/api/v1beta3/dynakube/extensions.go

```diff
@@ -6,8 +6,8 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+// +kubebuilder:validation:Optional
 type ExtensionsSpec struct {
-	// +kubebuilder:validation:Optional
 	Enabled bool `json:"enabled,omitempty"`
```

Comment:
`Enabled` flag is still in the spec here

Reply:
you could add some description on what the `extensions` section does as well!

Reply:
Oha, fixed in 37ac723